### PR TITLE
feedback

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
feedback

### CODE REVIEW

File: stories/assets/js/Tab_custom.js [hunk 1]
```diff
@@ -1,16 +1,6 @@
 /* Tab JS start custom */
```
Comment (shishirphadke) [inline stories/assets/js/Tab_custom.js:1]: @harishkumar-srijan Change the existing comments to tabs.js library here.
Eg. 
```
/**
 * tabs.js library
 */

```

File: stories/assets/js/Tab_custom.js [hunk 2]
```diff
@@ -1,16 +1,6 @@
...
-    const tab_id = $(this).attr('data-tab');
-    $tab_item.removeClass('active');
-    $tab_content.removeClass('active');
-    $(this).addClass('active');
-    $(`#${tab_id}`).addClass('active');
-  });
+   $(document).foundation();
+   $('.tabs li:first, .tabs-content .tabs-panel:first').addClass('is-active');
 }
 /* Tab JS end custom */
```
Comment (shishirphadke) [inline stories/assets/js/Tab_custom.js:6]: @harishkumar-srijan remove this comment.

File: stories/Components/UIcomponents/Tab/tab.scss [hunk 3]
```diff
@@ -17,32 +22,50 @@
...
+        border-bottom: 2px solid $color-dark-red;
+      }
+    }
+
     a {
       @extend %img_none;
-      
       font-size: $font-size-16;
+
+      &:focus, &:active, &:target, &:visited {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Components/UIcomponents/Tab/tab.scss]: done

File: stories/Components/UIcomponents/Tab/tab.scss [hunk 4]
```diff
@@ -17,32 +22,50 @@
...
+        background-color:$color-white;
+      }
+    }
+
+    a::selection {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Components/UIcomponents/Tab/tab.scss]: done

File: stories/assets/js/Tab_custom.js (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/assets/js/Tab_custom.js]: done

File: stories/assets/js/Tab_custom.js (hunk 2 above)
Comment (harishkumar-srijan) [reply stories/assets/js/Tab_custom.js]: done

File: stories/assets/js/Tab_custom.js [hunk 5]
```diff
@@ -1,16 +1,6 @@
 /* Tab JS start custom */
 export function TabFun() {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/assets/js/Tab_custom.js]: done

**Pull request merged**